src/styles/foundation/_focus-ring.scss

```diff
@@ -1,12 +1,17 @@
 /// Sets the focus ring for an interactive element
 /// @param {String} $size - The size of the border radius on the focus ring.
-/// @param {String} $offset - Focus ring offset border width + 1px.
+/// @param {String} $border-width - The border width of your element in rems.
```

Comment:
`@param {string}` doesn't seem to fit anymore. Is this a `Number` now?

Reply:
Yup, this would become `Number`